requirements.txt

```diff
@@ -28,6 +28,7 @@ setuptools>=67.0.0
 sigmatools==0.19.1 ; python_version > '3.4'
 six==1.17.0
 SQLAlchemy==1.4.48
+tqdm>=4.67.0
```

Comment:
The requirements are libs that need to be installed so that the timesketch server can run and will be installed in all build containers. As far as I can see, `tqdm` is only needed for the separate script in utils that will not be shipped with the containers.

So tracking the dependency in this file is probably the wrong place. I would either recommend to add a `requirements.txt` to the utils folder if necessary or mention to run `pip install tqdm` in the docs.

Reply:
yes